packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -226,16 +243,17 @@ function NoteEditor(props: Props, ref: any) {
 		// This variable is not used within this script
 		// but is called using "injectJavaScript" from
 		// the wrapper component.
-		let cm = null;
+		window.cm = null;
```

Comment:
Why do we need to change this? Can't it just be a regular variable?

Reply:
I was running into issues with `cm` being `undefined` in later calls to `webviewRef.current.injectJavaScript`. Making `cm` global in this way seemed to fix this. I'm confused as to why this wasn't an issue before (though perhaps it was?)...

Reply:
I think this change fixes #6374 ...

Using `let` instead of `window.` seems to not create issues on `iOS`, but it does seem to on Android...

Reply:
Hmm, ok, let's leave it as window.cm for now then. It doesn't seem so harmful either way.